include/flatbuffers/util.h

```diff
@@ -210,6 +210,9 @@ inline void EnsureDirExists(const std::string &filepath) {
   auto parent = StripFileName(filepath);
   if (parent.length()) EnsureDirExists(parent);
   #ifdef _WIN32
+    #ifdef _MSC_VER
+      #pragma warning(suppress: 6031) // "return value ignored: <function> could return unexpected value"
```

Comment:
Only the _mkdir call. That's why suppress is used instead of disable. Yes, prefixing `(void)` works, though I'm not sure I knew that was an option. I'll make the change.